src/Core/Framework/DataAbstractionLayer/Attribute/Field.php

```diff
@@ -17,7 +17,7 @@ public function __construct(
         public string $type,
         public bool $translated = false,
         public bool|array $api = false,
-        public ?string $column = null,
+        public ?string $storageName = null,
```

Comment:
this is a breaking change, as this is a public property. I am not sure about status of the whole attribute entity feature (if it is still declared as experimental or so), but it was already released and therefore, we need to consider this

Reply:
I'm hoping it is still marked as experimental as we've had enough issues with it to warrant several pull requests 😅

Also unsure of the state of this PR as I've had major merge conflicts with trunk that seemed to resolve the one of the issues I'm trying to solve in this PR. Maybe best to wait until Oliver is back from holiday to resolve all this.